docker-app/qfieldcloud/core/views/teams_views.py

```diff
@@ -0,0 +1,285 @@
+from django.core.exceptions import ObjectDoesNotExist
+from drf_spectacular.utils import extend_schema, extend_schema_view
+
+
+from qfieldcloud.core import permissions_utils
+from qfieldcloud.core.models import (
+    Organization,
+    TeamMember,
+    Team,
+)
+from qfieldcloud.core.serializers import (
+    TeamMemberSerializer,
+    TeamSerializer,
+)
+
+from rest_framework import generics, permissions
+
+from django.shortcuts import get_object_or_404
+
+
+class TeamMemberDeleteViewPermissions(permissions.BasePermission):
+    def has_permission(self, request, view):
+        user = request.user
+        organization_name = permissions_utils.get_param_from_request(
+            request, "organization"
+        )
+
+        try:
+            organization = Organization.objects.get(username=organization_name)
+        except ObjectDoesNotExist:
+            return False
+
+        if request.method == "DELETE":
+            return permissions_utils.can_delete_members(user, organization)
+
+        return False
+
+
+# @extend_schema_view(
+#     delete=extend_schema(description="Remove a member from a team"),
+# )
+# class TeamMemberDeleteView(generics.DestroyAPIView):
+#     permission_classes = [
+#         permissions.IsAuthenticated,
+#         TeamMemberDeleteViewPermissions,
+#     ]
+#     serializer_class = TeamMemberSerializer
+
+#     def get_queryset(self):
+#         """
+#         Define the queryset used for the view, ensuring objects are filtered
+#         based on URL parameters.
+#         """
+#         organization_name = self.kwargs.get("organization_name")
+#         team_name = self.kwargs.get("team_name")
+#         member_username = self.kwargs.get("member_username")
+
+#         return TeamMember.objects.filter(
+#             team__organization__username=organization_name,
+#             team__username=Team.format_team_name(organization_name, team_name),
+#             member__username=member_username,
+#         )
```

Comment:
Can you remove the commented code or add a comment why this is commented

Reply:
Done and also decorators have been added